packages/core/src/ide/ide-installer.ts

```diff
@@ -95,16 +94,12 @@ class VsCodeInstaller implements IdeInstaller {
     this.vsCodeCommand = findVsCodeCommand();
   }
 
-  async isInstalled(): Promise<boolean> {
-    return (await this.vsCodeCommand) !== null;
-  }
-
   async install(): Promise<InstallResult> {
     const commandPath = await this.vsCodeCommand;
     if (!commandPath) {
       return {
         success: false,
-        message: `VS Code command-line tool not found in your PATH or common installation locations.`,
+        message: `VS Code CLI not found. Please ensure 'code' is in your system's PATH. For help, see https://code.visualstudio.com/docs/configure/command-line#_code-is-not-recognized-as-an-internal-or-external-command. You can also install the companion extension manually from the VS Code marketplace.`,
```

Comment:
You may want to make a goo.gle link for the VSCode instructions + one for the yet-to-be-created companion extension in the marketplace